packages/core2/src/managers/BaseManager.ts

```diff
@@ -46,18 +51,23 @@ export abstract class BaseManager {
 	protected get user(): UserManager {
 		return this._sliceMachineManager.user;
 	}
-	protected get project(): ProjectManager {
-		return this._sliceMachineManager.project;
+	protected get repositoryAPI(): RepositoryAPIManager {
```

Comment:
"RepositoryAPI" doesn't sound right. It couples the manager to the HTTP API's endpoints rather than the more general idea of managing the repository. Anything to do with repositories should go in there, not just actions supported by the API.

`repository` and `RepositoryManager` sounds better to me.

Reply:
Will rename `prismicRepository`